addStyles.js

```diff
@@ -128,18 +130,30 @@ function removeStyleElement(styleElement) {
 
 function createStyleElement(options) {
 	var styleElement = document.createElement("style");
-	styleElement.type = "text/css";
+	options.tagAttrs.type = "text/css";
+
+	attachTagAttrs(styleElement, options.tagAttrs);
 	insertStyleElement(options, styleElement);
 	return styleElement;
 }
 
 function createLinkElement(options) {
 	var linkElement = document.createElement("link");
-	linkElement.rel = "stylesheet";
+	options.tagAttrs.rel = "stylesheet";
+
+	attachTagAttrs(styleElement, options.tagAttrs);
 	insertStyleElement(options, linkElement);
 	return linkElement;
 }
 
```

Comment:
It might not hurt to throw a quick `// document this function` in there given it's an optional feature.